include/gatekeeper_lls.h

```diff
@@ -211,6 +211,16 @@ struct lls_config {
 };
 
 /*
+ * TODO Enable an lcore to subscribe multiple times for the same IP address.
+ * This is needed because it's very hard for the GK blocks avoid duplicates.
+ * Currently, the LLS block has trouble with unsubscribing those duplicates
+ * since the LLS block identifies what to remove using
+ * the pair <IP_address, lcore_id>.
+ * In addition, the LLS block might not accept a subscription because
+ * it saves memory for a single subscription for lcore id.
+ */
+
+/*
```

Comment:
What are the side effects of this problem as a result of this patch? Will this problem cause bugs until this feature is added?